persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/PersistentEntityActor.scala

```diff
@@ -201,6 +201,12 @@ private[lagom] class PersistentEntityActor(
 
     case PersistentEntityActor.Stop =>
       context.stop(self)
+
+    /* The actor will receive a SaveSnapshotSuccess message when a snapshot
+       * was successfully saved. There's no need to do anything, but if we don't
+       * handle it, it will pollute the event stream with UnhandledMessage
+       * notifications */
```

Comment:
The indentation of the comment looks a little strange (but the comment itself is very much appreciated!)